### PR TITLE
Initial scaffolding

### PR DESCRIPTION
**Pull request by tortmayr**
- Maven module setup
- Common libraries as dependencies
- Inital setup for unit testing
- License information
- .travis.yml configuration (Should work once Travis is activated for this repository)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+### Eclipse Patch ###
+# Eclipse Core
+.project
+
+# JDT-specific (Eclipse Java Development Tools)
+.classpath
+
+# Annotation Processing
+.apt_generated
+/target/
```
Comment (edgarmueller) [inline .gitignore]: Add IDEA specific files as well: `.idea` and `*.iml`

**Review by edgarmueller — COMMENTED** [review]
Thanks for setting this up! Just a minor suggestion: Can we add some basic info about how to build/run/test this via Maven to the README?

**Comment by tortmayr** [comment]
Thanks for the review. I updated the gitignore file and the read-me.
Travis CI is now working as well

**Review by edgarmueller — APPROVED** [review]
Great, thank you!

**Pull request merged**